### DISCUSSION TITLE
MAUI for Windows Phone

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/maui#8507] [Ideas]
Now that we have MAUI, maybe microsoft's windows phone can be a good thing to think about? Sure the support was dropped however it is still up there because of the added transparency on the start menu tiles, let's make some app for bitcoin & NFT & cloud gaming for developers who still has the hardware then maybe we can bring it back, who knows what might happen.

At least if no one is using Windows phone this time, yet, MAUI can still run on other platforms

Let's try it with Lumia 930 because it was the newest model, we can tell other people how amazing this phone is once we can write apps for it, I love C# and I love .NET and I am going to be so happy when I write a program and it just works on all platforms.

We gonna play AAA games on our Lumia with cloud gaming, we gonna write programs using visual studio on Lumia with Github CodeSpace, we gonna move the heavy works to the cloud and partner with 5G operators and Azure to provide a new era of edge computing, powered by microsoft, what you desire turns into reality, powered by the reliable names of Microsoft products, from people who wants to provide an experience to the delivery of a frictionless new era, you can publish a game to the microsoft store or other providers, who has the capability to do cloud rendering, and you can include the cost of these services for example 300 Nvidia GeforceNow hours when people buy your games so they can start playing your games immediately on any devices anywhere, this benefits mobile device greatly. Of course applications can access local resources, however cloud resources becomes a cheap alternative to buying new hardware. Mobile manufactures can focus on the form and the feels and the looks of the devices, Microsoft can be the leaders again.

All we've been doing in the past couple years, with blazor server rendering and MAUI, helped me see this vision I really like Microsoft and what we are doing and I liked the idea of windows phone it has been a great time working on and using it and talking about these

**Comment by GreenShadeZhang** [comment]
I think you're a bad idea, wp itself is an outdated system that doesn't update, if you want to run maui I suggest you run Lumia 930 through a community project to run Windows 11, and then run maui.

**Reply by Xyncgas** [reply]
I thought maui doesn't depend on what kinds of system it is underlying, I thought MAUI cross platform

**Reply by GreenShadeZhang** [reply]
The windows implementation of MAUI relies on winui, but winui has minimum system requirements, so wp is not supported, you can use xamarin uwp.

**Reply by GreenShadeZhang** [reply]
Of course, it is irrational to support maintaining an unsupported system, and you can start another community project to do this.

**Comment by xtuzy** [comment]
windows phone have died because microsoft, your love to it is unworthy 🤣
